icefoganalytics/travel-authorization
# Refactor: Rename Travel Form Namespace to Travel Authorizations

# Context

The front-end namespace for content related to the "travel authorization" model was very ambiguous.
Now it matches the data modeling on the back-end. 
All UI stuff is the same, I've just renamed everything to make it easier to find stuff for the next developer, and less confusing for me.

# Testing Instructions

1. Boot the app via `dev up`
2. Log in to the app at http://localhost:8080
3. Go to the My Travel Requests page via the to nav.
4. Check that the table loads correctly.
5. Create a new travel authorizations, check that the page loads correctly.
6. Save the travel authorization as a draft, then go back to the table view via the "Back" button.
7. Check that the new travel authorization shows up in the table.
8. Create a new travel authorizations, but this time submit it to your supervisor.
9. Check that the "read only" view of the travel authorization loads correct.
10. Go back to the table view via the "Back" button, check that the submitted travel authorization also shows up correctly in the table.
